config/overlays/community/controller_manager_config.yaml

```diff
@@ -4,7 +4,8 @@ distribution: community
 health:
   healthProbeBindAddress: :8081
 metrics:
-  bindAddress: 127.0.0.1:8080
+  secure: true
```

Comment:
If `secure` is `true`, I assume we also need to set `--metrics-tls-cert-file`, is this set somewhere in the community version?

Reply:
When you don't specific certs those will be self-signed

Reply:
ok :+1: